docs/src/content/docs/challenges/angular/1-projection.md

```diff
@@ -32,6 +32,6 @@ While the application works, the developer experience is far from being optimal.
 ## Constraints
 
 - You <b>must</b> refactor the `CardComponent` and `ListItemComponent`.
-- The `NgFor` directive must be declared and remain inside the `CardComponent`. You might be tempted to move it to the `CardComponent` but avoid doing so.
+- The `NgFor` directive must be declared and remain inside the `CardComponent`. You might be tempted to move it to the `ListItemComponent` but avoid doing so.
```

Comment:
Indeed that's a mistake, but that's not the `ListItemComponent`, you must be tempted to move it to the `ParentCardComponent` like `TeacherCardComponent`, ...

Reply:
Fixed, thank you for your feedback:
<img width="951" alt="Screenshot 2023-10-29 at 16 31 21" src="https://github.com/tomalaforge/angular-challenges/assets/51910160/fad9326e-8cc0-4f93-8c4a-d2bc9aec15f8">